src/Essentials/src/DeviceInfo/DeviceInfo.ios.tvos.watchos.cs

```diff
@@ -45,6 +47,8 @@ static DeviceIdiom GetIdiom()
 		{
 #if __WATCHOS__
             return DeviceIdiom.Watch;
+#elif MACCATALYST || __MACCATALYST
```

Comment:
Should `__MACCATALYST` instead be `__MACCATALYST__`?

Reply:
Yes looks like a typo, however it shouldn't matter, the `MACCATALYST` is the proper one for net6, and `__MACCATALYST__` was really for legacy TFM's plus VSMac didn't recognize the `MACCATALYST` one until very recent internal builds.